integration_tests/e2e/plan-summary.cy.ts

```diff
@@ -0,0 +1,70 @@
+import CreateGoal from '../pages/create-goal'
+import PlanSummary from '../pages/plan-summary'
+
+describe('View Plan Summary', () => {
+  const planSummary = new PlanSummary()
+  const createGoalPage = new CreateGoal()
+
+  beforeEach(() => {
+    cy.createSentencePlan().then(planDetails => {
+      cy.wrap(planDetails).as('planDetails')
+      cy.openSentencePlan(planDetails.oasysAssessmentPk)
+      cy.visit('/plan-summary?source=nav')
+    })
+  })
+
+  it('Creates three new goals, and moves the middle goal up', () => {
+    createGoalPage.createCompleteGoal(0)
+    createGoalPage.createCompleteGoal(1)
+    createGoalPage.createCompleteGoal(2)
+    planSummary.clickUpOnSummaryCard(1)
+
+    planSummary
+      .getSummaryCard(0)
+      .should('contain', 'Accommodation Goal 2')
```

Comment:
I don't like that we're testing the title based on the autocomplete options here - we know these will change. Can you set the goal title with a parameter in "createCompleteGoal" and then test for that value here? I think it would also make the test clearer.

Reply:
I will address this.